client/comms/wsconn.go

```diff
@@ -360,19 +372,6 @@ func (conn *wsConn) keepAlive(ctx context.Context) {
 	}
 }
 
-// reconnect begins reconnection immediately.
-func (conn *wsConn) reconnect() {
-	conn.setConnected(false)
-	conn.reconnectCh <- struct{}{}
-}
-
-// queueReconnect queues a reconnection attempt.
-func (conn *wsConn) queueReconnect(wait time.Duration) {
-	conn.setConnected(false)
-	log.Infof("Attempting reconnect to %s in %d seconds.", conn.cfg.URL, wait/time.Second)
-	time.AfterFunc(wait, func() { conn.reconnectCh <- struct{}{} })
-}
```

Comment:
Using these from outside `read()` would be all kinds of problematic, so they are moved into `read`.